pyiron_base/project/maintenance.py

```diff
@@ -69,6 +72,37 @@ def get_repository_status():
         return report
 
 
+class LocalMaintenance:
+    def __init__(self, project):
+        self._project = project
+
+        from pyiron_base import Project
+
+        self._project = Project()
```

Comment:
Why do we need to do this?

Reply:
Thanks! This should not be in here... I just used it such that pycharm knows how to autocomplete... Typing hints would be of course better but result in circular import 
```suggestion
```